MetaMask/metamask-extension
# Enable zkSync Era in Swaps

## Explanation
This PR enables zkSync Era in Swaps.

TODO:
- Fix an issue with unapproved ERC20 to ERC20 swaps (backend).

## Screenshots/Screencaps
![image](https://github.com/MetaMask/metamask-extension/assets/80175477/4ab325a2-95ae-4659-a9c8-43abd3a6b375)

## Manual Testing Steps
- Run the extension locally via this command: `SWAPS_USE_DEV_APIS=1 yarn start`
- Add the zkSync Era network
- Check Swaps
